scene/gui/graph_edit.cpp

```diff
@@ -1999,6 +1999,9 @@ void GraphEdit::gui_input(const Ref<InputEvent> &p_ev) {
 		} else if (p_ev->is_action("ui_copy", true)) {
 			emit_signal(SNAME("copy_nodes_request"));
 			accept_event();
+		} else if (p_ev->is_action("ui_cut")) {
```

Comment:
Other actions pass `true` to the second parameter here, is there a reason why this one would need to differ?

Reply:
No, it seems to be a mistake on my part, will update it.

Reply:
For reference this is the check for exact match